src/hooks/firebase.hooks.jsx

```diff
@@ -21,10 +21,10 @@ export const useCreateUser = () => {
 
   return useMutation({
     mutationFn: (user) => {
-      addDoc(collection(db, 'users'), user);
+      addDoc(collection(db, 'users-v2'), user);
     },
```

Comment:
Consider add constants file or env file to save those things, 1 typo and it start to break